artifacts/camara-cloudevents/subscription-notification-template.yaml

```
@@ -0,0 +1,759 @@
+      allOf:
+        - $ref: "#/components/schemas/AccessTokenCredential"
+        - type: object
+          properties:
+            refreshToken:
+              description: REQUIRED. An refresh token credential used to acquire access tokens.
+              type: string
+            refreshTokenEndpoint:
+              type: string
+              format: uri
+              description: REQUIRED. A URL at which the refresh token can be traded for an access token.
+      required:
+        - refreshToken
+        - refreshTokenEndpoint
+
+    SubscriptionDetail:
+      description: The detail of the requested event subscription
+      type: object
+
+    CreateSubscriptionDetail:
+      description: The detail of the requested event subscription
+      type: object
+      required:
+        - type
+
+    XxxNotificationEventType:
+      type: string
+      description: |
+        xxx - Event triggered when a xxx event occured
+
+        subscription-ends: Event triggered when the subscription ends
+      enum:
+        - org.camaraproject.xxx.v0.xxx
+        - org.camaraproject.xxx.v0.subscription-ends
+
+    Subscription:
+      description: Represents a xxx subscription.
+      required:
+        - startsAt
+        - subscriptionId
+      allOf:
+        - $ref: '#/components/schemas/SubscriptionRequest'
+        - type: object
+          properties:
+            id:
+              type: string
+              description: The unique identifier of the subscription in the scope of the subscription manager.
+              example: 1119920371
+            status:
+              type: string
+              description: current status of the subscription - Management of subscriptionstate engine is not mandatory for now.
+              enum: ["REQUESTED", "ACTIVE", "EXPIRED", "DEACTIVATED"]
+          required:
+            - id
+
+    SubscriptionAsync:
+      description: Response for a xxx subscription request managed asynchronously (Creation or Deletion)
+      type: object
+      properties:
+          id:
+              $ref: '#/components/schemas/SubscriptionId'
+
+    SubscriptionId:
+      type: string
+      description: The event subscription identifier.
+      example: qs15-h556-rt89-1298
+
+    CloudEvent:
+      description: The notification callback
+      required:
+        - id
+        - source
+        - specversion
+        - type
+        - data
+      properties:
+        id:
+          type: string
+          description: identifier of this event, that must be unique in the source context.
+        source:
+          $ref: '#/components/schemas/Source'
+        type:
+          $ref: '#/components/schemas/XxxNotificationEventType'
+        specversion:
+          type: string
+          description: Version of the specification to which this event conforms (must be 1.0 if it conforms to cloudevents 1.0.2 version)
+          example: "1.0"
+          enum: ["1.0"]
```

It is syntactically correct, but more common style is used in https://github.com/camaraproject/Commonalities/pull/187
```suggestion
          enum: 
            - "1.0"
```